src/main/java/org/mitre/hapifhir/SMARTServerCapabilityStatementProvider.java

```diff
@@ -71,16 +74,21 @@ public CapabilityStatement getServerConformance(
     return c;
   }
     
-  private static CapabilityStatementRestSecurityComponent buildSecurityComponent(String tokenAddr) {
+  private static CapabilityStatementRestSecurityComponent buildSecurityComponent(String tokenAddr, String registrationAddr) {
```

Comment:
At this point it's probably cleaner to just make this an instance method (ie, not static) so that we can just use the instance properties instead of passing them in. Probably should have been an instance method to begin with tbh but I do like static methods.

Reply:
Yea, especially since it seems like the instance already has these two addresses.....